gen_install.sh

```diff
@@ -10,17 +10,34 @@ sudo apt-get -qq update -y
 
 if(($(cat /etc/os-release | grep VERSION_ID|grep -o '".*"' | sed 's/"//g' | cut -c1-2 )==16));then
     ROS_VERSION="kinetic"
-    else
-    if(($(cat /etc/os-release | grep VERSION_ID|grep -o '".*"' | sed 's/"//g' | cut -c1-2 )==18)); then
+fi    
+if(($(cat /etc/os-release | grep VERSION_ID|grep -o '".*"' | sed 's/"//g' | cut -c1-2 )==18)); then
     ROS_VERSION="melodic"
 fi
+if(($(cat /etc/os-release | grep VERSION_ID|grep -o '".*"' | sed 's/"//g' | cut -c1-2 )==20)); then
+    ROS_VERSION="noetic"
 fi
 echo "installing for ros version: ${ROS_VERSION}"
 sudo apt-get install -qq -y ros-${ROS_VERSION}-desktop-full
 source /opt/ros/${ROS_VERSION}/setup.bash
 ## install rosmon, it would be weird for this to be in one of the packages:
 #sudo apt-get install ros-${ROS_VERSION}-rosmon
+if(($ROS_VERSION="kinetic"||$ROS_VERSION="melodic")); then
+    sudo apt-get -qq install -y python-rosdep 
+    sudo apt-get -qq install -y python-pip
+    pip install 'mutagen==1.43.0' --user -q
+    echo "INSTALLING ROSDEP DEPENDENCIES"
+    sudo apt-get -qq install python-rosdep -y
 
+
+fi
+if(($ROS_VERSION="noetic")); then
+    sudo apt-get -qq install -y python3-rosdep
+    sudo apt-get -qq install -y python3-pip
+    pip install 'mutagen' --user -q
+    echo "INSTALLING ROSDEP DEPENDENCIES"
+    sudo apt-get -qq install python3-rosdep -y
+fi
 sudo apt-get -qq install -y python-rosdep
```

Comment:
looks like this line should be removed

Reply:
Sorry, I forgot I comment it out when I made all the changes again